python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -8,28 +8,19 @@ type Props = {
   size: "compact" | "full";
 };
 
-const useStyles = createStyles(() => ({
-  executeButton: {
-    borderBottomLeftRadius: 0,
-    borderTopLeftRadius: 0,
-    borderTopRightRadius: 0,
-  },
-}));
-
 export default memo(function RunPromptButton({
   runPrompt,
   size,
   isRunning = false,
 }: Props) {
-  const { classes } = useStyles();
   return (
     <Button
       onClick={runPrompt}
       disabled={isRunning}
       p="xs"
       size="xs"
       fullWidth={size === "full"}
-      className={classes.executeButton}
+      className="primaryPinkButton"
```

Comment:
Can we make this `runPromptButton` ? It's going to be yellow in gradio, so having pink in the name is confusing

Reply:
We will use the same class name ('.runPromptButton') with different styles. This RunPromptButton code is going to be the same in gradio and local editor so we can't change the className on the component itself, we can only change the styles for that classname

Reply:
great